parallel-install/pkg/components/components_test.go

```diff
@@ -65,3 +65,48 @@ func Test_GetComponents(t *testing.T) {
 	require.Equal(t, 2, len(res), "Number of components not as expected")
 	require.Equal(t, "comp2", res[0].Name)
 }
+
+func Test_ReuseHelmValues(t *testing.T) {
+	// fake k8s with override ConfigMaps
+	k8sMock := fake.NewSimpleClientset(
+		&v1.ConfigMap{
```

Comment:
Is this really needed? Can't see that these values are verified in this test somewhere.